src/server/ApiMessage.cpp

```diff
@@ -8,18 +8,51 @@
 #include "ApiMessage.h"
 #include "DeviceComm.h"
 
-_CONSOLE_API_MSG::_CONSOLE_API_MSG() :
-    Complete{ 0 },
-    State{ 0 },
-    _pDeviceComm(nullptr),
-    _pApiRoutines(nullptr),
-    _inputBuffer{},
-    _outputBuffer{},
-    Descriptor{ 0 },
-    CreateObject{ 0 },
-    CreateScreenBuffer{ 0 },
-    msgHeader{ 0 }
+inline constexpr size_t structPacketDataSize = sizeof(_CONSOLE_API_MSG) - offsetof(_CONSOLE_API_MSG, Descriptor);

```

Comment:
I forgot which way you explained this -- does `inline constexpr` force a symbol to be created for this? If so: do we need one here?

Reply:
Thanks for catching this mistake! `internal` forces `external` linkage.
Yeah... it's... it's something special indeed.